### DISCUSSION TITLE
Cisco WLCs and APs

### DISCUSSION THREAD

**Discussion by Astrotokii** [ipspace/netlab#1550] [Features]
If we could make a vWLC Vagrant box and an AP vagrant box, would it be possible to implement CAPWAP since it uses layer 2 functionality? We wouldn't necessarily need hosts to be able to connect to the AP for the lab purposes, because I imagine a wireless connection in netlab would be hard if not impossible, we could still build and verify proper config of WLANs and traffic between an AP and vWLC; in theory.

**Comment by ipspace** [comment]
Of course. You could use a P2P link between the two, or a VLAN, or even transport that data over VXLAN or EVPN.

The only gotcha might be if WLC uses some reserved IEEE MAC addresses, in which case Linux bridges might complain. Apart from that, passing L2 data between two nodes connected to the same VLAN should work just fine.
